### ISSUE TITLE
Add view for Local Enterprise Partnerships

### ISSUE THREAD

**Issue by slowe** [odileeds/ukpowernetworks#7]
We need:

- [x] Mapping from LSOA to LEP
- [x] Get GeoJSON for LEPs (non-overlapping and clipped to UKPN licence area)

**Comment by slowe** [comment]
ONS generalised clipped boundaries for LEPs (2020) are at https://geoportal.statistics.gov.uk/datasets/local-enterprise-partnerships-april-2020-boundaries-en-bgc?geometry=-8.492%2C50.628%2C8.845%2C53.004

**Comment by patricklake2** [comment]
Have added a [LSOA 2011 to LEP 2020 lookup file] (1 to 1) (../blob/main/2020-DFES/data/lsoa2lep.json) based on a best-fit lookup.

**Comment by slowe** [comment]
To reduce file size, I've cut out rivers, simplified some estuaries, and simplified some bits of coastline for the LEP map.

**Comment by slowe** [comment]
I discovered that switching the the "full extent" boundaries from ONS gives a much simpler coastline around East Anglia particularly. It also removes the rivers that go inland a long way. So I've switched to these and then used QGIS to `simplify` them to 0.001 degrees. I then removed some extraneous white space from the resulting GeoJSON file. This brings the LEP file down to 146kB (from 0.98MB) and the LAD file down to 430kB (from 1.74MB).
